crates/node/src/interactive_cli/context.rs

```diff
@@ -60,8 +60,9 @@ enum Commands {
     },
     /// Invite a user to a context
     Invite {
-        /// The context to invite the user to
-        context: Alias<ContextId>,
+        /// The context to invite the user to (omit to use default context)
+        #[clap(long = "context", short = 'c')]
+        context: Option<Alias<ContextId>>,
```

Comment:
Hm, can we try using clap's `default_value` instead?

```suggestion
        /// The context to invite the user to (omit to use default context)
        #[clap(long, short, default_value = "default")]
        context: Alias<ContextId>,
```

what do you think?

Reply:
PS: if you did this, you wouldn't need a `resolve_alias` impl that defaults to `"default"`, let's just compare both approaches. I just like the `clap` approach since it's less complicated to implement.